tsconfig.json

```diff
@@ -5,7 +5,8 @@
     "module": "es2015",                     /* Specify module code generation: 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
     "lib": [
       "dom",
-      "es2015"
+      "es2015",
+      "es2017"
```

Comment:
i'll have to do a little digging to understand what the implications of this are.

just curious, what made you reach for it?

Reply:
I was not able to `array.includes` or `string.includes`. Adding es2017 worked.

Reply:
> Adding `es2017` worked.

Unfortunately we need to ask TypeScript to target `es5`. Adding `es2017` as an additional lib enables support for the features you want in TypeScript-land, but it also means that consumers of our lib would be required to bring their own polyfill to the party because TypeScript doesn't transpile modern JS the way babel does.

i can't believe i'm linking to a reddit thread as supporting evidence, but it confirmed my hunch.

* https://stackoverflow.com/questions/41173215/does-typescript-transpilation-handle-transpiling-es6-to-es5
* https://www.reddit.com/r/typescript/comments/8gh2rt/any_way_to_use_es78_syntax_in_typescript_without/

i feel you though. i've reached for `.includes()` myself more than once here too.